tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "es5",
+    "target": "es6",
```

Comment:
I changed the ts config compiler options to "ES6" so the typescript does not complain about this method: `[...new Set([...checkBoxes ,...allCurrentPageUuids])]`